notebooks/lesson06_model-interpretation.ipynb

```diff
@@ -0,0 +1,2088 @@
+{
```

Comment:
<p>still not a fan of the star imports :P IMO it obscures where stuff comes from. is there an advantage to it?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lewtun/dslectures/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>speed :) but yes, maybe not a great educational practice</p>

Reply:
<p>i've removed them</p>